app/controllers/application_controller.rb

```diff
@@ -3,13 +3,14 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
 
-  def current_user
-		if session[:user_id]
-			return User.find(session[:user_id])
-		else
-			return nil
-		end
-	end
+  def check_session
```

Comment:
The name of this method is not very descriptive as to what its purpose is.  Maybe:

```ruby
def current_user
```